ci_framework/roles/edpm_build_images/tasks/install.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Ensure directories are present
+  ansible.builtin.file:
+    path: "{{ cifmw_edpm_build_images_basedir }}/{{ item }}"
+    state: directory
+  loop:
+    - tmp
+    - artifacts/repositories
+
+- name: Ensure working dir exists
+  file:
+    path: "{{ cifmw_edpm_build_images_work_dir }}"
```

Comment:
We can drop this and use `cifmw_edpm_build_images_basedir /artifacts/edpm_images`. The logs will be automatically collected by ci-framework.